rules/okta_rules/okta_geo_improbable_access.py

```diff
@@ -98,9 +97,8 @@ def store_login_info(key, event):
                 }
             )
         ],
+        epoch_seconds=event.event_time_epoch() + timedelta(days=7).total_seconds(),
```

Comment:
notice we were casting these to an int in other spots, might be useful to keep that pattern going